wa-admin/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "start-docker": "npm install && ng serve --host 0.0.0.0 --port 4250 --poll 2000",
+    "start-docker": "npm install && ng serve --host 0.0.0.0 --port 4250 --poll 2000 --disable-host-check",
```

Comment:
Why is this change required? Mostly curious as I doubt it will break existing functionality in our docker containers.